packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Columns.js

```diff
@@ -90,6 +95,32 @@ const Columns = ({
               ? instructionsLabel
               : disabledInstructionsLabel}
           </span>
+          <div
+            id={'customize-columns-select-all'}
+            className={classNames(`${blockClass}__draggable-handleHolder`, {
+              ['-selected']: getVisibleColumnsCount() > 0,
```

Comment:
Can this class change from `-selected` to something like `${blockClass}__draggable-handleHolder--selected`?

Reply:
Replaced `className={classNames(`${blockClass}__draggable-handleHolder`, {
              ['-selected']: getVisibleColumnsCount() > 0,`
with `className={classNames({
              [`${blockClass}__draggable-handleHolder`]: getVisibleColumnsCount() === 0,
              [`${blockClass}__draggable-handleHolder-selected`]: getVisibleColumnsCount() > 0,
           })}`

Also adjusted selectors in `_draggableElement.scss` to match the class names